docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
       - AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
       - AWS_SECRET_KEY=${AWS_SECRET_KEY}
       - ENABLE_BACKUP_MANAGEMENT=1
+      - ENABLE_RBAC=${ENABLE_RBAC}
```

Comment:
```suggestion
      - ENABLE_AC=${ENABLE_AC}
```

Reply:
This is how it's in the requirements. The flag will be removed with GA.
I did not want to change it so QA and docs is not confused.

Reply:
Yeah, but since we changed the naming shouldn't we change the environment variable as well? seems like it was forgotten during migration from RBAC to AC

Reply:
@maxkondr any thoughts about switching to `ENABLE_ACCESS_CONTROL`?